src/frontend/komponenter/Fagsak/Søknad/LeggTilBarn.tsx

```diff
@@ -124,10 +124,11 @@ const LeggTilBarn: React.FunctionComponent<IProps> = ({ settSøknadOgValider, s
                         <Knapp
                             type={'hoved'}
                             key={'Legg til'}
-                            mini
+                            mini={true}
```

Comment:
Tror ikke det er nødvendig å sette den til true, når det er en boolean som alltid skal være true, men det er kanskje smak og behag.

Reply:
Enig, jeg liker bare at det er eksplisitt. Siden man kan bruke shorthand for andre ting også så kan det være greit å være eksplisitt her tenker jeg.

Reply:
Skjønner 👍